runtime/doc/options.txt

```diff
@@ -3552,6 +3552,18 @@ A jump table for the options with a short description can be found at |Q_op|.
 	  eob		EndOfBuffer		|hl-EndOfBuffer|
 	  lastline	NonText			|hl-NonText|
 
+						*'findprg'* *'fprg'*
+'findprg' 'fprg'	string	(default "")
+			global or local to buffer |global-local|
+	External program to use for the |:find| command.  When this option is
+	empty the internal file finding mechanism is used.  When executing the
+	external command, "%" in 'findprg' is replaced with the current file
+	name.  To avoid this, escape "%" using "%%".
+	If a matching file is found, the command should return one or more
+	file names separated by the newline character.  Example:
+>
+	    :set findprg=find\ .\ -name\ '%'
```

Comment:
I don't quite understand the example. Why would anyone add the _current_ file name to `findprg`?

Reply:
The external find command needs a name to search for.  The "%" in 'findprg' is replaced with the argument passed to the ":find" command.

Reply:
So it's a placeholder like the `$*` in `grepprg` and `makeprg`. Shouldn't we use the same here to be consistent?